features/org.wso2.carbon.appmgt.core.feature/src/main/resources/sql/postgresql.sql

```diff
@@ -1,5 +1,6 @@
 BEGIN TRANSACTION;
-	
+
+DROP TABLE IF EXISTS IDN_BASE_TABLE;
```

Comment:
Having "DROP TABLE IF EXISTS" is not safer in dbscript with -Dsetup. In case -Dsetup has not properly handled to execute the script, this can result in loosing existing data if someone accidentally  start the server with -Dsetup later time. Suggestion is to use "CREATE TABLE IF NOT EXIST" or relevant statement for PostgreSQL.